app/frontend/static/views/partials/search.html

```diff
@@ -53,9 +76,9 @@
   <br/>
   <div ng-repeat="topic in results.docs" class="row">
     <div class="col-md-9">
-      <h4><a href="{{topic.listing_url}}">{{topic.title}}</a></h4>
-      <p>{{topic.description|truncate:500}}<span ng-if="topic.description.length > 500"><a href="{{topic.listing_url}}">MORE >>></a></span></p>
-      <button class="btn btn-default btn-sm pull-right" ng-click="saveOpportunity(topic.id)" type="button">Save for later</button>
+      <h4><a href="topic/{{topic.id}}">{{topic.title}}</a></h4>
```

Comment:
it's actually `topics/{{topic.id}}` - even for the single-item endpoint.  At least, that's how the API spec was written, so I wrote it that way... don't know if it makes sense.  Maybe HAL calls for it?